<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsMenu</class>
 <widget class="QWidget" name="OptionsMenu"> <!-- 改为 QWidget -->
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options Menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="homeButton">
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="powerOffButton">
     <property name="text">
      <string>Power Off</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createButton">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
